docs/experimental-features/host-risk-score.asciidoc

```diff
@@ -73,7 +73,7 @@ TIP: If you receive an error message during the installation process, delete the
 [discrete]
 === Upgrade host risk score 
 
-If you previously enabled host risk score and are upgrading to {stack} version 8.5 or later, there'll be an **Upgrade** button for host risk score instead of **Enable**. 
+If you previously enabled host risk score and are upgrading to {stack} versions 8.5–8.10, there'll be an **Upgrade** button for host risk score instead of **Enable**. If you're upgrading to {stack} version 8.11 or newer, you can upgrade to the latest risk scoring engine.
```

Comment:
I'm kinda confused: it sounds like regardless of the user's version, they can upgrade (we say there's an Upgrade button, or just that they can upgrade). It's not clear why we're distinguishing 8.5-8.10 from 8.11+ here.

Reply:
My understanding is that if they're upgrading from an earlier version to 8.5-8.10, they will still be using the 'original' (i.e. old) risk engine. Whereas if they're upgrading to 8.11, they can upgrade to the 'latest' (new) risk engine, for which the upgrade steps are different (or they can continue to run the old one if they want).
@SourinPaul, could you confirm if this is correct?

Reply:
> My understanding is that if they're upgrading from an earlier version to 8.5-8.10, they will still be using the 'original' (i.e. old) risk engine. Whereas if they're upgrading to 8.11, they can upgrade to the 'latest' (new) risk engine, for which the upgrade steps are different (or they can continue to run the old one if they want).

@natasha-moore-elastic, your understanding above is correct.

Reply:
@natasha-moore-elastic 💡 Will listing the two scenarios (stack version 8.5- 8.10 & stack version 8.11) as a bulleted list, improve the readability/ comprehension?

Reply:
Thanks for explaining and confirming, @natasha-moore-elastic & @SourinPaul. 

I think part of the problem is we're using 8.11 docs to document 8.5-8.10 functionality & behavior. For 8.11 docs, we don't need to explain what happens if a user upgrades _within the 8.5-8.10 range_, which is what that first sentence is talking about. We only need to explain what happens if they upgrade _to 8.11 or later_.

If there's different behavior depending on which version they're upgrading _from_, a bulleted list would be a good way to clearly distinguish the two scenarios.

Reply:
I think in the interest of time, let's focus on directions for upgrading to `8.11` or newer, then we'll tackle how to refer to upgrading in previous doc versions.